pepper/libpepper.py

```diff
@@ -361,6 +358,27 @@ def runner(self, fun, **kwargs):
 
         return self.low([low], path='/')
 
+    def wheel(self, fun, arg=None, **kwargs):
+        '''
+        Run a single command using the ``wheel`` client
+
+        Usage::
+          wheel('key.accept', match='myminion')
+        '''
+        low = {
+            'client': 'wheel',
+            'fun': fun,
+        }
+
+        if arg:
+            low['arg'] = arg
+        # if kwarg:
```

Comment:
The [`wheel` module(s)](https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html#module-salt.wheel.key) do take `kwargs` so I should include this right?